### DISCUSSION TITLE
Textarea - Specify Rows

### DISCUSSION THREAD

**Discussion by ainsleyclark** [payloadcms/payload#6404] [Feature Requests & Ideas]
It would be good if we could specify the amount of rows in a textarea field. 

```
{
    name: 'content',
    type: 'textarea',
    label: 'Content',
    admin: {
        rows: 10,
    }
}
```

**Comment by StefanEndress** [comment]
When adding adding lengthy text to a textarea, it grows infinitely in height. Would be great to add a max height via line amount.
